src/main/java/de/samply/share/client/util/connector/BrokerConnector.java

```diff
@@ -693,12 +693,12 @@ public void reply(InquiryDetails inquiryDetails, ISamplyResult result)
       throws BrokerConnectorException {
     ReplyEntity replyDonor = new ReplyEntity();
     replyDonor.setLabel("Donors");
-    replyDonor.setCount(NumberDisguiser.getDisguisedNumber(result.getNumberOfPatients()));
+    replyDonor.setCount(LaplaceMechanism.privatize(result.getNumberOfPatients(), 1, 0.12));
```

Comment:
Maybe introduce a config variable for epsilon?

Reply:
In addition, maybe introduce a config variable for the sensitivity and a comment, that the sensitivity equals one for counting queries.